cartography/models/ontology/mapping/data/containers.py

```diff
@@ -0,0 +1,90 @@
+from cartography.models.ontology.mapping.specs import OntologyFieldMapping
+from cartography.models.ontology.mapping.specs import OntologyMapping
+from cartography.models.ontology.mapping.specs import OntologyNodeMapping
+
+aws_ecs_container_mapping = OntologyMapping(
+    module_name="aws",
+    nodes=[
+        OntologyNodeMapping(
+            node_label="ECSContainer",
+            fields=[
+                OntologyFieldMapping(
+                    ontology_field="id", node_field="arn", required=True
+                ),
+                OntologyFieldMapping(ontology_field="name", node_field="name"),
+                OntologyFieldMapping(ontology_field="image", node_field="image"),
+                OntologyFieldMapping(
+                    ontology_field="image_digest", node_field="image_digest"
+                ),
+                OntologyFieldMapping(ontology_field="status", node_field="last_status"),
+                OntologyFieldMapping(ontology_field="cpu", node_field="cpu"),
+                OntologyFieldMapping(ontology_field="memory", node_field="memory"),
+                OntologyFieldMapping(ontology_field="region", node_field="region"),
+                # namespace: Not applicable for ECS containers (AWS does not use namespaces)
+                OntologyFieldMapping(
+                    ontology_field="health_status", node_field="health_status"
+                ),
+            ],
+        ),
+    ],
+)
+
+kubernetes_mapping = OntologyMapping(
+    module_name="kubernetes",
+    nodes=[
+        OntologyNodeMapping(
+            node_label="KubernetesContainer",
+            fields=[
+                OntologyFieldMapping(
+                    ontology_field="id", node_field="id", required=True
+                ),
+                OntologyFieldMapping(ontology_field="name", node_field="name"),
+                OntologyFieldMapping(ontology_field="image", node_field="image"),
+                OntologyFieldMapping(
+                    ontology_field="image_digest", node_field="status_image_sha"
+                ),
+                OntologyFieldMapping(
+                    ontology_field="status", node_field="status_state"
+                ),
+                # cpu: Not exposed as a direct field in KubernetesContainer node
+                # memory: Not exposed as a direct field in KubernetesContainer node
```

Comment:
Huh this would be super useful to have. Unrelated to this PR, let's add these feels to KubernetesContainers